test/interpreter/function-gamma.inv.spec.ts

```diff
@@ -0,0 +1,55 @@
+import {HyperFormula} from '../../src'
+import {ErrorType} from '../../src/Cell'
+import {ErrorMessage} from '../../src/error-message'
+import {adr, detailedError} from '../testUtils'
+
+describe('Function GAMMA.INV', () => {
+
+  it('should return error for wrong number of arguments', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=GAMMA.INV(1, 2)'],
+      ['=GAMMA.INV(1, 2, 3, 4)'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqualError(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+    expect(engine.getCellValue(adr('A2'))).toEqualError(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+  })
+
+  it('should return error for arguments of wrong type', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=GAMMA.INV("foo", 2, 3)'],
+      ['=GAMMA.INV(0.5, "baz", 3)'],
+      ['=GAMMA.INV(0.5, 2, "baz")'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqualError(detailedError(ErrorType.VALUE, ErrorMessage.NumberCoercion))
+    expect(engine.getCellValue(adr('A2'))).toEqualError(detailedError(ErrorType.VALUE, ErrorMessage.NumberCoercion))
+    expect(engine.getCellValue(adr('A3'))).toEqualError(detailedError(ErrorType.VALUE, ErrorMessage.NumberCoercion))
+  })
+
+  it('should work', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=GAMMA.INV(0.5, 1, 1)'],
+      ['=GAMMA.INV(0.9, 2, 4)'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toBeCloseTo(0.693147180559945, 6)
+    expect(engine.getCellValue(adr('A2'))).toBeCloseTo(15.5588806794697, 6)
+  })
+
+  it('checks bounds', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=GAMMA.INV(0, 1, 1)'],
+      ['=GAMMA.INV(-0.00001, 1, 1)'],
+      ['=GAMMA.INV(1, 1, 1)'],
+      ['=GAMMA.INV(0, 0, 1)'],
+      ['=GAMMA.INV(0, 1, 0)'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqual(0)
```

Comment:
Is this a bounding condition if it returns a correct value?

Reply:
yes, it shows that 0 is the minimal accepted value. I can remove it if you want it without this value.

Reply:
No, don't remove it. Just thought that we keep correctly calculated values in a different test "works"

Reply:
For me this is also 'respect bounds' type of test -- that bounds are '>=' not '>'